agent.go

```diff
@@ -32,15 +31,20 @@ func NewAgent(cfg *agentConfig) *Agent {
 		}
 		dm := newDeviceManager(dev.Name, dev.MTU, urls)
 		if err := dm.Run(); err != nil {
-			log.Printf("Error starting device `%s`: %v", dm.Name(), err)
+			logger.Error.Printf(
+				"Error starting device `%s`: %v",
+				dm.Name(),
+				err,
+			)
 			continue
 		}
 		agent.deviceManagers = append(agent.deviceManagers, dm)
 	}
 	tokenDir := filepath.Dir(defaultTokenFileLoc)
 	err := os.MkdirAll(tokenDir, 0755)
 	if err != nil {
-		log.Printf("Error: unable to create directory=%s", tokenDir)
+		logger.Error.Printf(
+			"Error: unable to create directory=%s", tokenDir)
```

Comment:
Do you still need `Error:` if its part of the logging object now?

Reply:
good catch!

Reply:
fixed